src/main/java/it/gov/pagopa/pu/debtpositions/repository/DebtPositionRepository.java

```diff
@@ -0,0 +1,10 @@
+package it.gov.pagopa.pu.debtpositions.repository;
+
+import it.gov.pagopa.pu.debtpositions.model.DebtPosition;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+
+@RepositoryRestResource(collectionResourceRel = "debt_position", path = "debt-position")
+public interface DebtPositionRepository extends JpaRepository<DebtPosition,Long> {
+
```

Comment:
let's provide a findById using the @EntityGraph as documented here:
https://blog.ippon.tech/boost-the-performance-of-your-spring-data-jpa-application (paragraph "Definition of the graph on the entity")
```suggestion

   @EntityGraph(value = "completeDebtPosition")
   Article findOneWithAllDataById(Long debtPositionId);

```